influxdb/client.py

```diff
@@ -101,6 +107,49 @@ def __init__(self,
             'Content-type': 'application/json',
             'Accept': 'text/plain'}
 
+    @staticmethod
+    def from_DSN(dsn):
```

Comment:
Good idea, I'll get that implemented this evening or tomorrow.

On Wed, Apr 8, 2015, 20:26 Alexandre Viau <notifications@github.com> wrote:

> In influxdb/client.py
> <https://github.com/influxdb/influxdb-python/pull/153#discussion_r28004098>
> :
>
> > @@ -101,6 +107,49 @@ def __init__(self,
> >              'Content-type': 'application/json',
> >              'Accept': 'text/plain'}
> >
> > +    @staticmethod
> > +    def from_DSN(dsn):
>
> I would make it def from_DSN(dsn, **kwargs): and pass all kwargs to the
> InfluxDBClient init.
>
> That way people can configure things like timeout and verify_ssl.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/influxdb/influxdb-python/pull/153/files#r28004098>.
>